<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventSelectionDialog</class>
 <widget class="QDialog" name="EventSelectionDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meggie - Epoch Creation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="leftMargin">
      <number>600</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCreateEpochs">
       <property name="text">
        <string>Create epochs</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>810</width>
        <height>451</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>741</width>
        <height>400</height>
       </size>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>0</y>
         <width>766</width>
         <height>444</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="labelName">
                  <property name="text">
                   <string>Event name:      </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditName"/>
                </item>
               </layout>
              </item>
              <item row="0" column="1" rowspan="2">
               <layout class="QVBoxLayout" name="verticalLayout"/>
              </item>
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="labelEventID">
                  <property name="text">
                   <string>Event ID:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBoxEventID"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButtonAdd">
                <property name="text">
                 <string>Add to list &gt;&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonRemove">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>&lt;&lt; Remove</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="labelCollectionName">
                <property name="text">
                 <string>Collection name:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditCollectionName">
                <property name="text">
                 <string>Epochs</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="labelTmin">
                  <property name="text">
                   <string>Start time:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBoxTmin">
                  <property name="suffix">
                   <string> s</string>
                  </property>
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="minimum">
                   <double>-10.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="value">
                   <double>-0.200000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="labelTmax">
                  <property name="text">
                   <string>End time:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBoxTmax">
                  <property name="suffix">
                   <string> s</string>
                  </property>
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="maximum">
                   <double>9.900000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="value">
                   <double>0.500000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QCheckBox" name="checkBoxMag">
                  <property name="text">
                   <string>mag</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxGrad">
                  <property name="text">
                   <string>grad</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxEeg">
                  <property name="text">
                   <string>eeg</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxStim">
                  <property name="text">
                   <string>stim</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxEog">
                  <property name="text">
                   <string>eog</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="labelRejections">
              <property name="text">
               <string>Rejection parameters</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_30">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_31">
                  <item>
                   <widget class="QLabel" name="labelGradReject_3">
                    <property name="text">
                     <string>Grad:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBoxGradReject_3">
                    <property name="prefix">
                     <string/>
                    </property>
                    <property name="suffix">
                     <string> fT/cm</string>
                    </property>
                    <property name="maximum">
                     <double>1000000000.000000000000000</double>
                    </property>
                    <property name="value">
                     <double>3000.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_32">
                  <item>
                   <widget class="QLabel" name="labelEegReject_3">
                    <property name="text">
                     <string>EEG:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBoxEEGReject_3">
                    <property name="suffix">
                     <string> uV</string>
                    </property>
                    <property name="maximum">
                     <double>1000000000.000000000000000</double>
                    </property>
                    <property name="value">
                     <double>40.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_33">
                  <item>
                   <widget class="QLabel" name="labelMagReject_3">
                    <property name="text">
                     <string>Mag:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBoxMagReject_3">
                    <property name="suffix">
                     <string> fT</string>
                    </property>
                    <property name="maximum">
                     <double>1000000000.000000000000000</double>
                    </property>
                    <property name="value">
                     <double>4000.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_34">
                  <item>
                   <widget class="QLabel" name="labelEogReject_3">
                    <property name="text">
                     <string>EOG:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBoxEOGReject_3">
                    <property name="suffix">
                     <string> uV</string>
                    </property>
                    <property name="maximum">
                     <double>1000000000.000000000000000</double>
                    </property>
                    <property name="value">
                     <double>250.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="listWidgetEvents">
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="pushButtonSaveEvents">
              <property name="text">
               <string>Save events to file...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonReadEvents">
              <property name="text">
               <string>Read events from file...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>comboBoxEventID</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonRemove</tabstop>
  <tabstop>lineEditCollectionName</tabstop>
  <tabstop>doubleSpinBoxTmin</tabstop>
  <tabstop>doubleSpinBoxTmax</tabstop>
  <tabstop>checkBoxMag</tabstop>
  <tabstop>checkBoxGrad</tabstop>
  <tabstop>checkBoxEeg</tabstop>
  <tabstop>checkBoxStim</tabstop>
  <tabstop>checkBoxEog</tabstop>
  <tabstop>doubleSpinBoxGradReject_3</tabstop>
  <tabstop>doubleSpinBoxMagReject_3</tabstop>
  <tabstop>doubleSpinBoxEEGReject_3</tabstop>
  <tabstop>doubleSpinBoxEOGReject_3</tabstop>
  <tabstop>pushButtonSaveEvents</tabstop>
  <tabstop>pushButtonReadEvents</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonCreateEpochs</tabstop>
  <tabstop>listWidgetEvents</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCreateEpochs</sender>
   <signal>clicked()</signal>
   <receiver>EventSelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>754</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>EventSelectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFileChooserDialog()</slot>
 </slots>
</ui>
